.gitignore

```diff
@@ -22,3 +22,5 @@ favicon.gif
 /public/favicon.gif
 .tool-versions
 .php-cs-fixer.cache
+
+composer.phar
```

Comment:
Let's leave this - if we ever have to use composer in the shared hosting, we have to have the composer.phar due to some limitations

Reply:
Should be zero changes to .gitignore for this PR - out of scope for the feature you are trying to develop